cmd/fablab/subcmd/clean.go

```diff
@@ -59,6 +59,9 @@ func clean(_ *cobra.Command, _ []string) {
 			}
 		} else {
 			logrus.Warnf("error loading label for instance [%s] (%v)", instanceId, err)
+			if err := model.RemoveInstance(instanceId); err != nil {
+				logrus.Fatalf("error removing instance [%s] (%v)", instanceId, err)
+			}
```

Comment:
Definitely don't think we should call `RemoveInstance` on an instance that we're not able to load a label from?

Reply:
I did that because it was a way to eradicate the dangling instance dir for the case I mentioned in the issue #64. The changes I made to create.go should now prevent/mitigate that potential, so I can remove this code if necessary.

Reply:
I removed the `RemoveInstance`

Reply:
Making sure that the model exists will help to avoid that situation in the future. The `~/.fablab` tree isn't meant to be a completely _sealed_ system, so it's okay to break out an editor or `rm -rf` to manually clean some of that stuff up if you do get into a weird state. But it could be a bad thing if a user ends up with an instance that was just created by a different version of `fablab`, but is fill of useful data... and we whack it just because that `fablab` build doesn't contain the referenced model.